src/core/qgis.cpp

```diff
@@ -46,6 +46,48 @@ const int Qgis::QGIS_VERSION_INT = VERSION_INT;
 
 // Release name
 const QString Qgis::QGIS_RELEASE_NAME( QStringLiteral( RELEASE_NAME ) );
+// Gdal Version string at build/compile time
+QString Qgis::GDAL_BUILD_VERSION( QString::fromUtf8( GDAL_RELEASE_NAME ) );
+const int Qgis::GDAL_BUILD_VERSION_MAJOR = GDAL_VERSION_MAJOR;
+const int Qgis::GDAL_BUILD_VERSION_MINOR = GDAL_VERSION_MINOR;
+const int Qgis::GDAL_BUILD_VERSION_REV = GDAL_VERSION_REV;
+QString Qgis::GDAL_RUNTIME_VERSION = QString::null;
+int Qgis::GDAL_RUNTIME_VERSION_MAJOR = -1;
+int Qgis::GDAL_RUNTIME_VERSION_MINOR = -1;
+int Qgis::GDAL_RUNTIME_VERSION_REV = -1;
+int Qgis::GDAL_OGR_RUNTIME_SUPPORTED = 0;
+bool Qgis::ogrRuntimeSupport()
+{
+  // Called in QgisApp and tested in QgsOgrProvider constructor
+  if ( Qgis::GDAL_RUNTIME_VERSION.isNull() )
+  {
+    // do this only once
+    Qgis::GDAL_RUNTIME_VERSION = QString( "%1" ).arg( GDALVersionInfo( "RELEASE_NAME" ) );
+    // Remove non-numeric characters (with the excetion of '.') : '2.2.0dev' to '2.2.0'
+    QString s_GdalVersionInfo = Qgis::GDAL_RUNTIME_VERSION;
```

Comment:
QGIS standard naming is camel case, with the "s" prefix reserved for static variables